Doc/Tutorial/chapter_quick_start.tex

```diff
@@ -140,6 +140,8 @@ \subsection{Simple GenBank parsing example}
 
 You'll notice that a shorter string has been used as the \verb|seq_record.id| in this case.
 
+Actually the FASTA file is a little old fashioned as the NCBI has deprecated the old numeric GI scheme.
+
```

Comment:
Does it matter?

Reply:
It was in part a note to self, we may want to update the FASTA file to match current NCBI style.